app/src/components/common/CardSignInRequired.vue

```diff
@@ -23,21 +23,6 @@
           title="Ledger Nano"
           route="/ledger"
         />
-        <LiSession
-          v-if="!isMobileApp"
-          id="use-extension"
-          icon="laptop"
-          title="Lunie Browser Extension"
-          route="/extension"
-        >
-        </LiSession>
-        <LiSession
-          v-if="accountExists && (session.insecureMode || isMobileApp)"
-          id="sign-in-with-account"
-          icon="lock"
-          title="Sign in with account"
```

Comment:
i know this was in the issue but i don't remember why. what's the reason @faboweb ? how does a user use their existing local accounts on mobile?

Reply:
they appear in the user menu